kubernetes/services/api-gateway.yaml

```diff
@@ -43,6 +38,16 @@ data:
     keystone.password=alcor_pass
     keystone.auth_type=password
     keystone.auth_url=http://192.168.10.10:5000/v3
+    neutron.url_prefix=/v2.0
+    logging.level.root=TRACE
+    
+    Spring.mvc.async.request-timeout=30000
+    server.max-http-header-size=65536
+    server.max-initial-line-length=65536
+    
+    spring.sleuth.sampler.probability=1.0
+    spring.application.name=api-gateway
```

Comment:
@kevin-zhonghao Please remove `spring.sleuth.sampler` and `spring.application.name`.